rest/src/test/java/eu/europa/ec/fisheries/uvms/activity/rest/resources/FishingActivityResourceTest.java

```diff
@@ -138,6 +141,236 @@ public void listActivityReportsByQuery_noSearchCriteria() throws JsonProcessingE
         }
     }
 
+    @Test
+    public void listActivityReportsByQuery_multipleActivityTypes() throws JsonProcessingException {
+        // Given
+        FishingActivityQuery query = new FishingActivityQuery();
+
+        Map<SearchFilter, List<String>> searchCriteriaMapMultiVal = new HashMap<>();
+        List<String> activityTypeValues = new ArrayList<>();
+        activityTypeValues.add("FISHING_OPERATION");
+        activityTypeValues.add("DEPARTURE");
+        searchCriteriaMapMultiVal.put(SearchFilter.ACTIVITY_TYPE, activityTypeValues);
+        searchCriteriaMapMultiVal.put(SearchFilter.PURPOSE, Lists.newArrayList("9"));
+        query.setSearchCriteriaMapMultipleValues(searchCriteriaMapMultiVal);
+
+        // When
+        String responseAsString = getWebTarget()
+                .path("fa")
+                .path("list")
+                .request(MediaType.APPLICATION_JSON)
+                .header(HttpHeaders.AUTHORIZATION, authToken)
+                .header("scopeName", null)
+                .header("roleName", "myRole")
+                .post(Entity.json(query), String.class);
+
+        // Then
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        PaginatedResponse<FishingActivityReportDTO> responseDto =
+                objectMapper.readValue(responseAsString, new TypeReference<PaginatedResponse<FishingActivityReportDTO>>() {
+                });
+
+        assertEquals(200, responseDto.getCode());
+        assertNull(responseDto.getMsg());
+
+        List<FishingActivityReportDTO> resultList = responseDto.getResultList();
+
+        assertEquals(41, responseDto.getTotalItemsCount());
+        assertEquals(41, resultList.size());
```

Comment:
Maybe loop over them and see that all have one of the two activity types we put in the query.